tests/test_dirichlet_distributions.py

```diff
@@ -348,3 +348,48 @@ def test_geodesic_vectorization(self):
         result = geod(time).shape
         expected = (self.n_points, self.dim)
         self.assertAllClose(expected, result)
+
+    @geomstats.tests.np_only
+    def test_jac_christoffels(self):
```

Comment:
Nice! It would be great to test if the value of the jacobian is what is expected on a single well-chosen point, i.e. not only testing the shape and the vectorization but also the result itself.

I see that you added `@geomstats.tests.np_only`: what are the problems for tf and pytorch, did you try running it with these backends?

Reply:
That's what I did at first using the ```jacobian``` from autograd. But then I needed to add a ```jacobian``` function to the autograd file of the other backends, and I didn't know how. If you have insights on this, I am interested !

Reply:
@nguigs any idea there?

Reply:
Yes, but where did you need it? I don't see any call to gs.autograd.jacobian in the PR?

Reply:
In both pytorch-tensorflow/autograd files, a function `jacobian` needs to be added that behaves like the one from the `autograd` package.
For tensorflow, it should look like:
```python
def jacobian(f, x):
    if isinstance(x, np.ndarray):
        x = tf.Variable(x)
    with tf.GradientTape() as g:
      g.watch(x)
      y = f(x)
    return g.jacobian(y, x)
```
Here is a discussion on how to do it with pytorch: https://gist.github.com/sbarratt/37356c46ad1350d4c30aefbd488a4faa

Reply:
If you can show your original code using `jacobian` for the numpy backend I can make sur the above solutions work